sample-app/.gitignore

```diff
@@ -29,6 +29,11 @@ build/
 .gradle
 local.properties
 *.iml
+*/**/fabric.properties
```

Comment:
Don't add anything here we don't expect our consumers to add. You can add this to your global gitignore if need be, but don't pollute the sample app please

Reply:
Removed fabric.properties